.github/workflows/build.yml

```diff
@@ -23,7 +23,6 @@ on:
       compiler-flags:
         description: "Compiler Flags - e.g. 'DEBUG_MENU FEATURE2'"
         type: string
-        default: "DEBUG_MENU"
```

Comment:
🤔 just want to make sure this change is intended?

Reply:
I would think so @vvolkgang could you confirm?

Reply:
We need it in main but for visibility, let's revert it from this merge and I'll open a PR with this change targeting main.

Reply:
Btw nice catch Phil and it has been reverted Álison 😄